website/content/docs/agent/options.mdx

```diff
@@ -642,18 +642,33 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
   - `tokens` ((#acl_tokens)) - This object holds all of the configured
     ACL tokens for the agents usage.
 
-    - `master` ((#acl_tokens_master)) - Only used for servers in the [`primary_datacenter`](#primary_datacenter).
+    - `initial_management` ((#acl_tokens_initial_management)) - Only used for
+      servers in the [`primary_datacenter`](#primary_datacenter). This token will
+      be created with management-level permissions if it does not exist. It allows
+      operators to bootstrap the ACL system with a token Secret ID that is well-known.
+
+      The `initial_management` token is only installed when a server acquires cluster
+      leadership. If you would like to install or change it, set the new value for
+      `initial_management` in the configuration for all servers. Once this is done,
+      restart the current leader to force a leader election. If the `initial_management`
+      token is not supplied, then the servers do not create an initial management token.
+      When you provide a value, it should be a UUID. To maintain backwards compatibility
+      and an upgrade path this restriction is not currently enforced but will be in a
+      future major Consul release.
+
+    - `master` ((#acl_tokens_master)) **Deprecated in Consul 1.11. Use the
+      [`acl.tokens.initial_management`](#acl_tokens_initial_management) field insead.** -
+      Only used for servers in the [`primary_datacenter`](#primary_datacenter).
       This token will be created with management-level permissions if it does not
       exist. It allows operators to bootstrap the ACL system with a token Secret
       ID that is well-known.
 
-      The `master` token is only installed when a server acquires cluster
-      leadership. If you would like to install or change the `acl_master_token`,
-      set the new value for `master` in the configuration for all servers. Once
-      this is done, restart the current leader to force a leader election. If
-      the `master` token is not supplied, then the servers do not create a
-      master token. When you provide a value, it should be a UUID. To maintain
-      backwards compatibility and an upgrade path this restriction is not
+      The `master` token is only installed when a server acquires cluster leadership.
+      If you would like to install or change it, set the new value for `master` in the
+      configuration for all servers. Once this is done, restart the current leader to
+      force a leader election. If the `master` token is not supplied, then the servers
+      do not create a master token. When you provide a value, it should be a UUID. To
+      maintain backwards compatibility and an upgrade path this restriction is not
```

Comment:
Why repeat the text from `initial_management`? Can this just reference rather than copy? Similar to how [`-non-voting-server` is handled](https://www.consul.io/docs/agent/options#_non_voting_server)?

e.g.,
```suggestion
    - `master` ((#acl_tokens_master)) **Deprecated in Consul 1.11. See the
      [`acl.tokens.initial_management`](#acl_tokens_initial_management) field instead.** -
```

Reply:
Ah, nice! Copied this approach from some of the other deprecated ACL fields (e.g. [`acl_datacenter`](https://www.consul.io/docs/agent/options#acl_datacenter)) but agree just referencing is much bettert.